test/rules-google-enhanced-ecommerce-fullstory.spec.ts

```diff
@@ -68,7 +68,7 @@ describe('Ruleset: Google Analytics Enhanced Ecommerce to FullStory', () => {
         let [eventName, payload] = await testHarness.popEvent();
         expectEqual(eventName, 'detail_product');
         expectEqual(payload.id, 'P000525722');
-        expectEqual(payload.price, 2.99);
+        expectEqual(payload.price, '2.99');
```

Comment:
In the previous ruleset version in this repository, a convert operator converted these numeric strings to numbers. For data layer integrations, that conversion is handled by a `beforeDestination` operator. Since we're testing the ruleset in isolation, these values aren't converted. In the future, it may be worth testing these rulesets + expected `beforeDestination` operators to more completely test data layer integration behavior.